site/app/templates/admin/admin_gradeable/AdminGradeablePeers.twig

```diff
@@ -313,7 +313,7 @@
     * @param {String} button_id the ID of the submit button to be enabled / disabled
     * @param (String) error_div_id the ID of the error message to be displayed if submit is disabled
     */
-    function updateSubmitButton(valid_grader, valid_students, new_grader, grader_removed, button_id, error_div_id) {
+    function updateSubmitButton(valid_grader, valid_students, new_grader, grader_removed, button_id, error_div_id, self_grading) {
```

Comment:
This function confuses me, as every instance that I can see of it being called uses a `false` value for self_grading, is this the intended behavior?

Reply:
yeah we need some change , actually we are changing the self_grading value inside callback function, it return true (when the student and grader is same) but outside the function it return the global value(which is false)

currently it return the same error message i will fix that and then add test for that